mailgun/resource_mailgun_domain_test.go

```diff
@@ -185,7 +210,7 @@ func testAccDomainCheckDestroy(domain *fullDomain) resource.TestCheckFunc {
 		mg := testAccProvider.Meta().(*mailgun.MailgunImpl)
 		ctx, cancel := context.WithTimeout(context.Background(), time.Second*30)
 		defer cancel()
-
+		time.Sleep(5 * time.Second)
```

Comment:
does other terraform provider use hard coded `sleep`ing times like this?

Reply:
I don't know i will check.

Reply:
I didn't found one in a test like that but i found one elsewhere.